src/main/resources/assets/traverse/lang/fr_fr.json

```diff
@@ -0,0 +1,59 @@
+{
+  "itemGroup.traverse.items": "Objets de Traverse",
+
+  "biome.traverse.arid_highlands": "Hautes terres arides",
+  "biome.traverse.autumnal_wooded_hills": "Collines automnales",
```

Comment:
I think the wooded_hills translations should mirror the English phrasing. This just says "autumnal hills" instead of "autumnal wooded hills," which seems to have a different meaning.

Also, I'm just curious why automnales is used here yet automnaux is used for the non-hills variant. Google Translate translates "automnales" as "fall" instead of "autumnal" so I'm just curious if there's a difference in meaning.

Reply:
French has nouns with genders. "Colline" is a feminine word therefore is its adjective. The other nouns that use "automnaux" as their adjective are masculine words.